sippy-ng/src/tests/TestTable.js

```diff
@@ -685,7 +740,7 @@ function TestTable(props) {
         autoHeight={true}
         rowHeight={100}
         disableColumnFilter={props.briefTable}
-        disableColumnMenu={true}
+        disableColumnMenu={false}
```

Comment:
I would suggest leaving this true.  The column menu settings doesn't get saved in links so it's harder to share data.  The `views` feature covers this by having static views stored in the URL.